image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -5,29 +5,39 @@
     state: enforcing
```

Comment:
the default driver remains devicemapper unless the user specifies overlay in their inventory, correct?